databroker/assets/handlers.py

```diff
@@ -142,6 +142,8 @@ class HDF5DatasetSliceHandler(HandlerBase):
         key of the single HDF5 Dataset used by this Handler
     frame_per_point : integer, optional
         number of frames to return as one datum, default 1
+     swmr : bool, optional
```

Comment:
Is there a stray space before `swmr` here? That will probably break sphinx formatting.